doc/nrf/ug_zigbee.rst

```diff
@@ -0,0 +1,137 @@
+.. _ug_zigbee:
+
+Working with Zigbee
+###################
+
+The |NCS| provides support for developing applications using the Zigbee protocol.
+
+.. _zigbee_ug_intro:
+
+Introduction
+************
+
+Zigbee is a portable, low-power software networking protocol that provides connectivity over 802.15.4-based mesh network.
+It also defines an application layer that provides interoperability among all vendors.
+
+In combination with Zephyr RTOS and |NCS|, Zigbee allows for easy development of low-power connected solutions.
+
+.. _zigbee_ug_supported features:
+
+Supported features
+******************
+
+|zboss_lib|
+It includes all mandatory features of the |zigbee_version| specification and provides an Application Programming Interface to access different services.
+The stack comes with the following features:
+
+* Complete implementation of the Zigbee core specification and Zigbee Pro feature set.
+* Support for all device roles: coordinator, router, and end device.
+* Zigbee Cluster Library (a specification for end devices).
+* Base Device Behavior.
+* Devices, described in former Zigbee Home Automation and Light Link profiles.
+* Zigbee Green Power.
+* Experimental support for ``ZB_ZCL_WWAH``.
+
+Required libraries and drivers
+******************************
+
+|zboss_lib|
+This library is available in `nrfxlib`_.
+
+The OSIF subsystem acts as the linking layer between the ZBOSS stack and |NCS|.
+It implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
+
+Configuration
+*************
+
+To use the Zigbee protocol, set the :option:`CONFIG_ZIGBEE` Kconfig option.
+Setting this option enables all the peripherals required for the correct operation of the Zigbee protocol and allows you to use them.
+
+After that, you have to define the Zigbee device role for the Zigbee application or sample by setting one of the following Kconfig options:
+
+* Router role: :option:`CONFIG_ZIGBEE_ROLE_ROUTER`
+* End device role: :option:`CONFIG_ZIGBEE_ROLE_END_DEVICE`
+* Coordinator role: :option:`CONFIG_ZIGBEE_ROLE_COORDINATOR`
+
+Setting any of these options enables the respective ZBOSS role library.
+This is needed because end devices use different libraries than routers and coordinators.
+
```

Comment:
You could link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#configuring-your-application here for instructions on how to set these config options.

Reply:
Done.